packages/appium/lib/main.js

```diff
@@ -399,8 +400,13 @@ async function main(args) {
     serverOpts.keepAliveTimeout = parsedArgs.keepAliveTimeout * 1000;
   }
   let server;
+  const bidiServer = new WebSocketServer({noServer: true});
```

Comment:
same as above. The web socket is expected to use SSL if the server already uses it for http communication

Reply:
I think I'll need help with this part. I'm just creating a virtual server object. It's not an actual server. I'm not sure we need to worry about ssl here since the server is already configured with ssl and this is just 'attaching' as an upgrade handler?